app/components/article-featured-video.js

```diff
@@ -38,12 +37,9 @@ export default Component.extend(InViewportMixin,
 			return !this.get('fastboot.isFastBoot') &&
 				!this.get('withinPortableInfobox') &&
 				this.get('supportsAutoplay') &&
-				$.cookie(autoplayCookieName) !== '0' &&
-				inGroup('MOBILE_FEATURED_VIDEO_AUTOPLAY', 'AUTOPLAY');
-		}),
-		autoplayToggleVisible: computed('supportsAutoplay', function () {
-			return this.get('supportsAutoplay') && inGroup('MOBILE_FEATURED_VIDEO_AUTOPLAY', 'AUTOPLAY');
+				$.cookie(autoplayCookieName) !== '0';
 		}),
+		autoplayToggleVisible: computed.oneWay('supportsAutoplay'),
```

Comment:
https://m.alphasights.com/the-risks-you-take-when-you-use-ember-computed-oneway-f2c008a14404

I think computed.reads is better here,